_posts/2016-08-17-decentralization-is-about-diversity.md

```diff
@@ -0,0 +1,27 @@
+layout: post
+title: "Decentralization is about diversity"
+author: Kenton Varda
+authorUrl: https://github.com/kentonv
+imageUrl: https://sandstorm.io/news/images/decentralize.png
+---
+
+![](/news/images/decentralize.png)
+
+Lately, there has been a flurry of activity around decentralizing the web. Summits have been held. New projects -- and companies -- have been started. Having worked on this issue for several years now, we're excited to see it becoming increasingly mainstream.
+
+But what, exactly, are we trying to solve? Most people think it has something to do with privacy, and maybe also security. Some argue that data ownership and mobility are the most important things. Sometimes this leads decentralization projects to focus on data storage while neglecting compute. Some projects even propose that so long as storage is decentralized, it doesn't matter where the software actually runs.
+
+Privacy, security, ownership, and mobility are all important, but I feel there is a much more important goal that is often poorly understood:
+
+<p style="font-weight: bold; text-align: center; font-size: 130%; margin: 1em auto; width: 500px; max-width: 90%;">The most important reason to decentralize is software&mdash;and developer&mdash;diversity.</p>
+
+By "diversity", I mean this: Who can develop software that other people can plausibly use? Is it primarily mega-corps like Apple and Google? VC-backed startups? Or can one random person, working in their spare time, build just the right app and reach millions of people? Can a community of unfunded volunteers build an open source app that wins because it is the best? Can an employee of one company, having built a useful internal app to solve a problem they had, give (or sell) that app to another company in the same position, without a lot of hassle? Can a teenage fan of a particular video game build an app that assists players of the game, and then share that app with the rest of the game's community?
+
+All of the above happens regularly with mobile and desktop apps, but it is far more difficult on the web.
+
+We live in a time when our tools are so good that a single competent application developer working weekends can create almost any web application you can imagine. However, rarely can that single developer also run a secure, scalable service and a business around it. As a result, when software is delivered as a service, the only software that is available is that which was deemed *a priori* to be sufficiently lucrative to interest a mega-corp like Google or a VC-backed startup. **Therefore, the only software we get comes courtesy of these gatekeepers.** Experimental, indie, or amateur projects are rare. Software serving a small niche community is rare. Software designed by people who aren't well-represented in the tech industry is rare. Community-driven open source projects <a href="https://sandstorm.io/news/2014-07-21-open-source-web-apps-require-federated-hosting">basically aren't viable</a>.
```

Comment:
This isn't strictly true:

> Experimental, indie, or amateur projects are rare. 

To be cruel but accurate, you could say instead:

> Experimental, indie, or amateur projects rarely last longer than a year or two online.

For now I'm just thinking out loud.